### PR TITLE
fix(client+server): add return cash only settlement action

### LINKED ISSUE #168: Define if a "return only cash" button is needed in payoff dialog
The use case would be to return money, but leave books there in case they could be sold later on
This is really an edge case and probably not actually needed

Estimated required time: ~4 hours

### PR DESCRIPTION
**Pull request by andrea-cerlini**
fixes #168

## What it does
This feature adds a new button to the payoff dialog actions to return the money of the unsettled sold books without returning/changing ownership of the books of the customer that are still in stock.

## How to test it
From the users management page, click on the checkout button for a user with sold books, then press the button to return cash only

### Additional notes
The label for the confirmation dialog may need further checks on the wording

### CODE REVIEW

File: packages/client/src/components/manage-users/pay-off-user-dialog.vue [hunk 1]
```diff
@@ -706,6 +708,33 @@ function returnAllBooks(remainingType: SettleRemainingType) {
...
+    try {
+      const { cache } = await settleUser({
+        input: {
+          remainingType: "CASH_ONLY",
+          retailLocationId: selectedLocation.value.id,
+          userId: props.user.id,
+        },
+      });
+
+      evictQuery(cache, GetSoldBookCopiesDocument);
```
Comment (IlCallo) [inline packages/client/src/components/manage-users/pay-off-user-dialog.vue:734]: Why is this required?
We're not removing sold books

I guess this was part of the previous implementation for which you hid the settled sales

File: packages/client/src/i18n/it/manage-users.ts [hunk 2]
```diff
@@ -144,6 +148,7 @@ export default {
       "Non è stato possibile segnalare il problema per tutte le copie.",
     returnAndDonate: "Restituisci contanti e dona libri",
```
Comment (IlCallo) [inline packages/client/src/i18n/it/manage-users.ts]: Why isn't there the amount to return here? I guess we should add it

@NiccoloMenozzi this seems something that has been decided into design phase, as it has been implemented as per mockup

"Return cash only" is the new button we added

Actual
image

Zeplin
image

**Review by IlCallo — CHANGES_REQUESTED** [review]

File: packages/client/src/components/manage-users/pay-off-user-dialog.vue (hunk 1 above)
Comment (andrea-cerlini) [reply packages/client/src/components/manage-users/pay-off-user-dialog.vue:734]: This is needed to update the local cache's status regardless, just like in `returnAllBooks` (even though there `refetch` is called instead); should I add the same comment, change it with a `refetch` call, or leave it as it is?

Comment (IlCallo) [reply packages/client/src/components/manage-users/pay-off-user-dialog.vue:734]: Let's leave it as is, it works, I'll try to better understand how it actually works later on

File: packages/client/src/i18n/it/manage-users.ts (hunk 2 above)
Comment (IlCallo) [reply packages/client/src/i18n/it/manage-users.ts]: I added the amount in all buttons

**Pull request merged**